science/hacktv/files/destdir-fix.diff

```diff
@@ -0,0 +1,27 @@
+--- Makefile.orig	2022-10-09 22:13:57.000000000 +0100
++++ Makefile	2022-10-09 22:41:52.000000000 +0100
+@@ -1,5 +1,5 @@
+ 
+-CC      := $(CROSS_HOST)gcc
++CC      := $(CROSS_HOST)$(MACPORTS_CC)
```

Comment:
why not use `$(CC)` here?

Reply:
Thus, you may keep `$(CROSS_HOST)gcc` as default value:
```Makefile
CC ?= $(CROSS_HOST)gcc
```

Reply:
I wanted to minimally alter the upstream person's makefile. I suppose I could have deleted line 5 altogether and it may have hand the same effect as what I did. But I just did the same thing this other way.  

There is an openmaintainer added, so you are welcome to change it if you wish.

Reply:
Ah - my last post must have crossed your one explaining:

`CC ?= $(CROSS_HOST)gcc`

which I have only just now seen.  Let me think about this.

Reply:
@kesterlester I usually backport to upstream all my patches which I did to my ports here. Near all of them usually merged :)

I'd like to suggest to do the same way here: adjust Makefile as you needed and backport it as PR.

If one day such PR is merged, you simple remove patch :)

Reply:
ok